src/cli.js

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 
 import argv from 'yargs';
 
-import { version } from 'json!../package';
```

Comment:
Yes, the JSON loader is loaded and used implicitly if there is none defined in the config: https://webpack.js.org/guides/migrating/#json-loader-is-not-required-anymore